version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.5.15-SNAPSHOT"
+ThisBuild / version := "0.5.16-SNAPSHOT"
```

Comment:
No need to bump the snapshot version here, the version just gets bumped as part of the release process

Reply:
Fixed